ComposableArchitecture.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
Should this file have been deleted? Wonder if it's needed for CI, but guess we'll find out!